dashboards/xdcr-dashboard.json

```diff
@@ -660,7 +660,31 @@
         {
           "datasource": "{data-source-name}",
           "expr": "sum by (status) (xdcr_pipeline_status{status=\"Running\"})",
-          "legendFormat": "{data-source-name} number of running replications}",
+          "legendFormat": "{data-source-name} {{targetClusterUUID}} number of running replications}",
+          "_base": "target"
+        }
+      ]
+    },
+    {
+      "title": "XDCR Number of total replications",
+      "_base": "panel",
+      "_targets": [
+        {
+          "datasource": "{data-source-name}",
+          "expr": "xdcr_number_of_replications_total",
+          "legendFormat": "{data-source-name} {{targetClusterUUID}} number of total replications per remote cluster}",
+          "_base": "target"
+        }
+      ]
+    },
+    {
+      "title": "XDCR Number of total replications decrease alert",
+      "_base": "panel",
+      "_targets": [
+        {
+          "datasource": "{data-source-name}",
+          "expr": "count(xdcr_number_of_replications_total < xdcr_number_of_replications_total offset 1m)",
+          "legendFormat": "{data-source-name} Number of replication decreased in the past minute, detail in \"XDCR Number of total replications\"}",
```

Comment:
My only comment is that the `legendFormat` is kind of long. I see what that you're trying to add descriptive information to help folk understand what the panel means. However, it can make the legend area below the graph a bit complicated and busy as this information could be repeated for all the data sources. I find the panel title to be a useful place to put this kind of information. There's also the `description` attribute that can be added to the panel which results in an `i` information icon getting added to the top-right hand corner of the panel where you can put descriptive text.

Reply:
Thanks. I should be able to move it to the panel title.

Reply:
Moved to the (i) via description field.

https://github.com/couchbaselabs/promtimer/compare/4a55530d6484e59b981724c4c775aa3e8fa7a85b..87b8589557565bb0da9e63f265b3ffc9d8d1a5be